lib/src/shared/main/java/com/couchbase/lite/internal/core/C4DocumentEnded.java

```diff
@@ -29,15 +29,15 @@
     private String revID;               // Referenced from native code
     private int flags;                  // Referenced from native code
     @SuppressFBWarnings("UUF_UNUSED_FIELD")
+    @SuppressWarnings("PMD.UnusedPrivateField")
     private long sequence;              // Referenced from native code
     private int errorDomain;            // Referenced from native code: C4Error.domain
     private int errorCode;              // Referenced from native code: C4Error.code
     private int errorInternalInfo;      // Referenced from native code: C4Error.internal_info
-    @SuppressWarnings("PMD.AvoidFieldNameMatchingMethodName")
     private boolean errorIsTransient;   // Referenced from native code:
 
     // Called from native code
-    public C4DocumentEnded() { }
+    // public C4DocumentEnded() { }
```

Comment:
intentional?

Reply:
mmmm....   Static analysis didn't like it.  I should probably just take it out...